### ISSUE TITLE
File unavailable on complete

### ISSUE THREAD

**Issue by juanpasolano** [balderdashy/skipper#72]
Every time I upload a file and the on-complete function is called I try to get the image but it is never available. Is like when the on-complete function is called the image has not been saved. If a require it after a little while it is available. 

Does anyone else has this behaviour?

**Comment by metzuda** [comment]
I run into this if I pass incorrect parameters when uploading to S3; but that's my problem.

**Comment by sailsbot** [comment]
Thanks for posting, @juanpasolano.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
